storage.js

```diff
@@ -0,0 +1,58 @@
+exports.storage = {
```

Comment:
This data object can be imported into the server. This would be quite nice as the data would be separate from code and keep server.js cleaner